src/changelog/unreleased.md

```diff
@@ -84,6 +84,7 @@ changelog entry.
 - On Web, `Window::canvas()` now returns a reference.
 - On Web, `CursorGrabMode::Locked` now lets `DeviceEvent::MouseMotion` return raw data, not OS
   accelerated, if the browser supports it.
+- On Android, when `keycode` can be represented as character, it is used as `KeyEvent::text`.
```

Comment:
```suggestion
- On Android, when `keycode` can be represented as character, it is provided in `KeyEvent::text`.
```

Or even:

```suggestion
- On Android, when `keycode` can be represented as character, its textual representation is provided in `KeyEvent::text`.
```

Should we also mention that it applies to logical and not physical keys, in case a mapping is active?

Reply:
I'm not that familiar with this stuff, so I need to count on you on this. :)

Reply:
Me neither, which is why I expect the other maintainers to chime in and confirm that this is a sensible change to make.

It's always a bit scary when neither contributor nor reviewer have a definitive answer :)

Reply:
That seems redundant with the `KeyEvent` documentation?